### ISSUE TITLE
CDC COVID-19 - Bug: Self-Checker section's description text does not meet color contrast requirements

### ISSUE THREAD

**Issue by tiffanyjc** [a11ySea/testing#2]
## Issue Summary

The description text in the "Self-Checker" and "Take Precautions" section on the CDC page has insufficient color contrast (4.35). 

## Steps to reproduce

1. Go to https://www.cdc.gov/coronavirus/2019-ncov/index.html
2. Scroll down to the Self-Checker and Take Extra Precautions section (they are adjacent)
3. The description text does not meet expected contrast ratio (but the header is OK)

### Expected behavior

Expected contrast ratio for this text size is at least 4.5:1. 

### Actual behavior

The description text in the "Self-Checker" section on the CDC page has insufficient color contrast of 4.35 (foreground color: #007d91, background color: #ebf5f6, font size: 15.0pt (20px), font weight: normal). Expected contrast ratio is 4.5:1. 

# Specifications

- **WCAG: WCAG 1.4.3**

## Additional References

- Link to any articles or specs that give further guidance
